### DISCUSSION TITLE
👋 Chào Mừng anh em đã đến với GitHub Discussion của FullstacKAGE 👋

### DISCUSSION THREAD

**Discussion by p-le** [p-le/fullstackage#1] [Thông báo!]
## 👋 Welcome! :tada: 

Mình sử dụng **GitHub Discussion** như một nơi để giao lưu với anh em. Hi vọng mọi người trao đổi vui vẻ tại đây.
  * Hỏi đáp Q/A
  * Share ý tưởng
  * Trao đổi với các thành viên khác
  * Cởi mở, nói chuyện văn minh lịch sự.

:rocket: Các discussion mới sẽ tự động notify về Slack Channel của mình, để mình có thể cập nhật và phản hồi sớm nhất có thể :rocket:

**Comment by luongduc11a3** [comment]
em chào anh ạ. Em làm phiền anh chút được không ạ. Chả là em vô tình xem được video của anh trên youtube nên em muốn tìm đến anh để học hỏi. Tuy rằng anh biết đấy em không mang lại tác dụng hay giá trị gì cho anh cũng ảnh hưởng đến thời gian công việc của anh nhưng nếu có thể em mong đc anh giúp đỡ a. Dù a có gíup hay ko em thì em cũng muốn cảm ơn anh nhiều và thật may mắn khi xem đc kênh của anh. Nó rất hữu ích và gíup đỡ nhiều ae đang theoo hướng cloud engineer. Mong rằng em xin đc a 1 chút thời gian để học hỏi ạ.
